expan/core/early_stopping.py

```diff
@@ -159,6 +165,8 @@ def get_or_compile_stan_model(model_file, distribution):
             pickle.dump(sm, f)
     return sm
 
+cacheSamplingResults = False
+samplingResults = {} # memoized sampling results
```

Comment:
According to PEP8:

Function Names
Function names should be lowercase, with words separated by underscores as necessary to improve readability.
mixedCase is allowed only in contexts where that's already the prevailing style, to retain backwards compatibility.

Method Names and Instance Variables
Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.
Use one leading underscore only for non-public methods and instance variables.

We can look which style is prevailing not to change much then.